common/djangoapps/student/tests/test_email.py

```diff
@@ -244,7 +244,7 @@ def setUp(self):
         self.course = None
         self.user = UserFactory()
 
-    @ddt.data('course_run_1', 'matt''s course', 'matt＇s run')
+    @ddt.data('course_run_1', 'matt''s course', 'matt＇s run')  # pylint: disable=implicit-str-concat
```

Comment:
Looking at [the commit that added this line](https://github.com/openedx/edx-platform/commit/ebbb2400278232b3433999538749a0ca823f4e96), I don't think this was meant to be implicit string concatenation, and the line should be fixed instead:
```py
@ddt.data('course_run_1', 'matt\'s course', 'matt＇s run')
```
@schenedx can you confirm?

Reply:
Updated as per the suggestion.